test/images/agnhost/Dockerfile

```diff
@@ -27,7 +27,7 @@ CROSS_BUILD_COPY qemu-QEMUARCH-static /usr/bin/
 # - iproute2: includes ss used in NodePort tests
 # from iperf image
 # install necessary packages: iperf, bash
-RUN apk --update add bind-tools curl netcat-openbsd iproute2 iperf bash && rm -rf /var/cache/apk/* \
+RUN apk --update add bind-tools curl nmap-ncat iproute2 iperf bash && rm -rf /var/cache/apk/* \
```

Comment:
in order to use this, you have to follow a process, it is described here
https://github.com/kubernetes/kubernetes/pull/94135#issuecomment-680037641

basically, you have to promote a new image first, and then submit a new PR with the test using it

Reply:
ok, shall I move forward with that?  wanted to make sure the PR overall is in good standing b4

Reply:
Why are we adding ncat if we use nc above?

Reply:
Ncat is the new one that has all the good stuff in it, like sctp.... I think this is what you want in your related Pr to add sctp functionality to the DialFromNode verifiers